Cargo.toml

```diff
@@ -8,6 +8,6 @@ repository = "https://github.com/rory/rust-wkb"
 license = "AGPL-3.0+"
 
 [dependencies]
-geo = "0.10.2"
```

Comment:
question, do you need the geo dependency ?

if possible I think it's better to depend on `geo-types`, you have less pain to update the dependency.

Reply:
`geo-types` would be fine as well, I'd imagine.

(I'm not actually sure whether I can submit further PRs for `wkb`. I recently noticed that uses an AGPL license, which I believe my employer prefers to avoid even in our open source projects.

This obviously isn't a problem on your end! Your library is a great contribution to the Rust community, and of course you should license it however you'd like. But I may not be be able to submit more PRs without talking to people.)